README.md

```diff
@@ -25,19 +25,15 @@ This app can be run locally using:
     $ poetry run flask run
 ```
 
-This project uses flask env variables for the Trello API
+This project uses flask env variables 
 These need to be added to a .env file and include:
 ```
-    * Key
-    * TOKEN
-    * BoardID
-    * ToDoId (List Id)
-    * DoingId (List Id)
-    * DoneId (List Id)
+    * Mongo_Url
+    * Mongo_db
 ```
 
 This project also uses Docker
-Docker production and development containers can be built using:
+Docker production and development containers can be built using, with a Port argument passed in:
```

Comment:
I can't say it's not there, but if this is the solution it would be nice to just update the command! i.e. change the run command to `docker run --env-file .env -p 5000:5000 -e PORT=5000 todo-app:prod`